### DISCUSSION TITLE
What about an integration of DCAT-AP?

### DISCUSSION THREAD

**Discussion by ticapix** [eclipse-edc/.github#43] [Q&A] [answered]
Hi,

Is it part of the roadmap to integrate [DCAT-AP](https://joinup.ec.europa.eu/collection/semantic-interoperability-community-semic/solution/dcat-application-profile-data-portals-europe) to increase semantic interoperability with existing European data catalogues ?

Best regards,
Pierre

**Comment by clange** [comment, marked as answer]
Yes, it is on the agenda. The IDS Information Model is already largely compatible with DCAT-AP; see https://github.com/International-Data-Spaces-Association/InformationModel/issues/277. A separate but also relevant process is cleaning up the information model towards direct reuse of DCAT terms rather than reinventing parts of it in the IDS namespace, due end of 2021. (@JohannesLipp could you please link the relevant issue(s) here?) Finally, DCAT-AP is one possibility to achieve compatibility with data portals. Another one is via the CKAN API, which has been implemented in a different connector by Fraunhofer FOKUS; see https://github.com/public-data-space/ids-open-data-connector.

**Comment by MoritzKeppler** [comment]
thx, Christoph!

**Comment by JohannesLipp** [comment]
FYI the relevant issues in the IDSA info model repository are:
https://github.com/International-Data-Spaces-Association/InformationModel/issues/461
https://github.com/International-Data-Spaces-Association/InformationModel/issues/490
